ci-operator/step-registry/baremetalds/metallb-e2e/test/baremetalds-metallb-e2e-test-commands.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
 
-set -o nounset
-set -o errexit
-set -o pipefail
+set -e
 
 echo "************ baremetalds metallb e2e test command ************"
 
 # shellcheck source=/dev/null
 source "${SHARED_DIR}/packet-conf.sh"
 
 echo "### Copying metallb directory"
+set +e
```

Comment:
Can you check if `/go/src/github.com/openshift/metallb` exists instead of swallowing the error?